data/json/monsters.json

```diff
@@ -1736,6 +1744,7 @@
     "armor_bash":6,
     "armor_cut":6,
     "vision_day":30,
+    "vision_night":2,
```

Comment:
OK, that's not obvious to this dev based on game experience, so I doubt players'd recognize it.  Might be worth a line in the description once 0.C hits.